### PR TITLE
Molecule fixes for idrac_certificate

### PR DESCRIPTION
**Pull request by ShivamSh3**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by rajshekarp87 — DISMISSED** [review]

File: roles/idrac_certificate/molecule/generateCSR/converge.yml [hunk 1]
```diff
@@ -29,11 +34,13 @@
           organization_name: "OrgName"
           subject_alt_name:
             - 192.198.2.1
```
Comment (rajshekarp87) [inline roles/idrac_certificate/molecule/generateCSR/converge.yml]: Use FQDN instead of IP

**Review by rajshekarp87 — CHANGES_REQUESTED** [review]

File: roles/idrac_certificate/molecule/generateCSR/converge.yml (hunk 1 above)
Comment (ShivamSh3) [reply roles/idrac_certificate/molecule/generateCSR/converge.yml]: done

File: roles/idrac_certificate/molecule/CTC/converge.yml [hunk 2]
```diff
@@ -4,51 +4,87 @@
...
+      register: ctc_cert_file
+      delegate_to: "{{ idrac_delegate_to }}"
+      no_log: true
 
-    - name: Verifying CTC import certificate
+    - name: Asserting operation with Normal/Idempotence mode.
       ansible.builtin.assert:
         that:
-          - idrac_certificate_out.msg == "No changes found to be applied." or idrac_certificate_out.msg == "Successfully performed the 'import' operation."
+          - ctc_cert_file.stat.exists
```
Comment (ABHISHEK-SINHA10) [inline roles/idrac_certificate/molecule/CTC/converge.yml:81]: Good 👍 that you have added other assertion, normally I see only for msg.

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Pull request merged**